src/client/components/ContactForm/ContactForm.component.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './ContactForm.styles.css';
+
+export default function ContactForm({user}) {
+    return (
+        <div className="infoBox">
```

Comment:
You have 3 containers: infoBox, wrapper and form. You could reduce it to one, because I think you have not assigned any css properties to  infobox nor wrapper. So, do you need them?

Reply:
Thanks for the review, code adjusted as suggested.